public/output.js

```diff
@@ -0,0 +1,140 @@
+// Open and connect output socket
+let socket = io('/output');
+
+// Listen for confirmation of connection
+socket.on('connect', function() {
+  console.log("Connected");
+});
+
+let users = {}; // track users
+let flipOrder[]; // track order users flipped their phones in
```

Comment:
Whoa does this (`[]`) work in javascript?

Reply:
Lol no, should be let flipOrder = [];